source/lib-intro.tex

```diff
@@ -486,8 +486,8 @@
 
 \pnum
 Several entities and \grammarterm{typedef-name}{s}
-defined in \ref{\firstlibchapter} through \ref{\lastlibchapter} and \ref{depr}
-are only defined for the purpose of exposition.
+defined or declared in \ref{\firstlibchapter} through \ref{\lastlibchapter} and \ref{depr}
+are only defined or declared for the purpose of exposition.
```

Comment:
"declared" alone is good enough; each definition is also a declaration.

Reply:
Done. [extern.types] is also changed.